pages/api/account/register.ts

```diff
@@ -7,20 +7,26 @@ import {
 import { NextApiRequest, NextApiResponse } from "next";
 import { middleware, cors, csrfProtected } from "@lib/middleware";
 import { isValidGovEmail } from "@lib/validation";
+import { ResponseErrors } from "@lib/types";
+import { ResponseStatus } from "@lib/types/response-errors";
 
 const register = async (req: NextApiRequest, res: NextApiResponse) => {
   const { COGNITO_REGION, COGNITO_APP_CLIENT_ID } = process.env;
 
   // craft registration params for the SignUpCommand
   if (!req.body.username || !req.body.password || !req.body.name) {
     return res.status(400).json({
```

Comment:
All response objects could also go through a util method but eh that could be annoying